### ISSUE TITLE
signup page for users who are not logged in is needed

### ISSUE THREAD

**Issue by sudhanshuprasad** [shubham7668/hacktober#252]

**Comment by piyush-mahajan** [comment]
I can make the page, I have past experience in it so can you please assign me this issue? I would love to work on this project.

**Comment by VINEETMISHRA0001** [comment]
sir can you assign this issue to me,  I will add both the beauty  and functionality to the form

**Comment by Yatndeep** [comment]
you can assign me this work

**Comment by guneshmunjal** [comment]
probably just bootstrap is needed and a bit of javascript to fix this issue and i can do that if you allow me to do so 
kindly assign me this under hactoberfest

**Comment by sudhanshuprasad** [comment]
Yep sure... But I am only a contributor like you. You may add that
otherwise I will do this 🙂

**Comment by guneshmunjal** [comment]
By the way how many PR you've done and got merged in hactoberfest? Sudhanshu

**Comment by Mohsinali0786** [comment]
I can do this task easily I have 2+ year experienced please brief me that you need only ui or functionality+ui

**Comment by sudhanshuprasad** [comment]
I've not contributed in hacktober fest yet but I have contributed a lot in
a few other open source projects
